package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -652,37 +740,46 @@ def run(self, **kwargs):
                     for h in donor_h_set:
                         res = ns_acceptors.search(h, self.distance)
                         for a in res:
-                            donor_atom = h if self.distance_type != 'heavy' else d
+                            donor_atom = h if self.distance_type != 'heavy' \
+                            else d
                             dist = self.calc_eucl_distance(donor_atom, a)
                             if dist <= self.distance:
                                 angle = self.calc_angle(d, h, a)
                                 if angle >= self.angle:
                                     self.logger_debug(
-                                        "WB-A: {0!s} <-> S2-D: {1!s} {2:f} A, {3:f} DEG".format(a.index, h.index, dist, angle))
-                                    if not (a.resname, a.resid) in s2_frame_results_dict:
-                                        s2_frame_results_dict[(a.resname, a.resid)] = []
-                                    s2_frame_results_dict[(a.resname, a.resid)].append(
+                                        "WB-A: {0!s} <-> S2-D: {1!s} {2:f} A, {3:f} DEG".format(a.index, h.index,
+                                        dist, angle))
+                                    if not (a.resname, a.resid) in \
```

Comment:
Please indent for legibility. Follow [PEP8 indentation rules](https://www.python.org/dev/peps/pep-0008/#indentation)